nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/cf/JMSConnectionFactoryProvider.java

```diff
@@ -97,7 +96,7 @@
             .description("Path to the directory with additional resources (i.e., JARs, configuration files etc.) to be added "
                     + "to the classpath. Such resources typically represent target MQ client libraries for the "
                     + "ConnectionFactory implementation.")
-            .addValidator(new ClientLibValidator())
+            .addValidator(StandardValidators.createListValidator(true, true, StandardValidators.createURLorFileValidator()))
```

Comment:
I don't think this is the proper validation here. Changing it to support a list of files/directories/urls is probably a good idea. However, at present this processor expects that the configured value be a directory. I think we need to either update the processor to accept the list, or otherwise just use `StandardValidators.createDirectoryExistsValidator(true, false);`

Reply:
Yep. Thanks for pointing out. I have modified the processor to accept a comma separated list of paths that can be added to Classpath and moreover it would leverage `ClassLoaderUtils` thereby avoiding duplicate.